core/store/ledgerstore/ledger_store.go

```diff
@@ -915,6 +915,12 @@ func (this *LedgerStoreImp) GetBlockRootWithNewTxRoots(startHeight uint32, txRoo
 	if this.currBlockHeight > startHeight+uint32(len(txRoots))-1 {
 		// or return error?
 		return common.UINT256_EMPTY
+	} else if this.currBlockHeight+1 < startHeight {
+		// this should never happen in normal case
+		log.Fatalf("GetBlockRootWithNewTxRoots: invalid param: curr height: %d, start height: %d",
```

Comment:
log.Fatalf seems no different with panic?

Reply:
change to log.Error?

Reply:
will not panic, just an higher log level.